hack/tools.go

```diff
@@ -1,13 +1,12 @@
 // +build tools
 
 /*
-Copyright 2020 The Knative Authors.
-
+Copyright 2020 The Knative Authors
```

Comment:
```suggestion
Copyright 2020 The Knative Authors

```

Reply:
I literally copy pasted this copyright from another file